manywheel/build_common.sh

```diff
@@ -162,11 +162,23 @@ else
     echo "BUILD_DEBUG_INFO was not set, skipping debug info"
 fi
 
+if [[ "$DISABLE_RCCL" = 1 ]]; then
+    echo "Disabling NCCL/RCCL in pyTorch"
+    USE_RCCL=0
+    USE_NCCL=0
+    USE_KINETO=0
+else
+    USE_RCCL=1
+    USE_NCCL=1
+    USE_KINETO=1
+fi
```

Comment:
@pruthvistony Should this patch be upstreamed?

Reply:
@jithunnair-amd ,
Recommend to keep this change in upstream, will be required to build pyTorch without NCCL/RCCL support and work through torch_UCC. On the code front, it is guarded by `DISABLE_RCCL` flag and should NOT affect any other builds.